bodybank/templates/home.html

```diff
@@ -5,12 +5,14 @@
 
 {% block content %}
 {% if user.is_authenticated %}
-  Hi {{ user.username }}!
-  <p><a href="{% url 'list_egos' %}">My Egos</a></p>
-  <p><a href="{% url 'logout' %}">Log Out</a></p>
+  <div class="container">
+    Hi {{ user.username }}!
+  </div>
 {% else %}
-  <p>You are not logged in</p>
-  <a href="{% url 'login' %}">Log In</a>
-  <a href="{% url 'signup' %}">Sign Up</a>  
+  <div class="container">
```

Comment:
the div is the same in if/else so can deduplicate it by moving it out of the if/else